src/main/java/com/dev/cinema/validation/FieldsValueMatch.java

```diff
@@ -0,0 +1,25 @@
+package com.dev.cinema.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = FieldsValueMatchValidator.class)
+@Target({ ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface FieldsValueMatch {
+    String message() default "Fields values don't match!";
+    Class<?>[] groups() default {};
+    Class<? extends Payload>[] payload() default {};
+    String field();
+    String fieldMatch();
+
+    @Target({ElementType.TYPE})
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface List {
+        FieldsValueMatch[] value();
```

Comment:
you can use this block if you check several fields through "repeat" like - password and repeatpassword, email and repeatemail. Otherwise you don't need those lines 20-23

Reply:
Yes, but but from what I can say this is just more general implementation. Due to the fact that validate can be applied not just for user and not just for password but for other fields and models as well. So I think in case of scailing it's quite possible to have need in check of several fields